### PR TITLE
precompile: Remove extra checks

### PR DESCRIPTION
**Pull request by mustermeiszer**
# Description

Dropping any EVM storage related validation that we have in place and treat this precompile as a simple forwarded for gateway messages.

# Checklist:

- [x] I have added Rust doc comments to structs, enums, traits and functions
- [x] I have made corresponding changes to the documentation
- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: pallets/liquidity-pools-gateway/axelar-gateway-precompile/src/cargo_expand.rs
Comment (mustermeiszer) [inline pallets/liquidity-pools-gateway/axelar-gateway-precompile/src/cargo_expand.rs:1]: ^^ 

I feel you pain.

Comment (cdamian) [reply pallets/liquidity-pools-gateway/axelar-gateway-precompile/src/cargo_expand.rs:1]: Hehe =D

File: pallets/liquidity-pools-gateway/axelar-gateway-precompile/src/lib.rs [hunk 1]
```diff
@@ -246,30 +245,16 @@ where
 	#[precompile::public("execute(bytes32,string,string,bytes)")]
 	fn execute(
 		handle: &mut impl PrecompileHandle,
-		command_id: H256,
```
Comment (cdamian) [inline pallets/liquidity-pools-gateway/axelar-gateway-precompile/src/lib.rs:249]: Should we drop this or keep as is to keep it consistent with the gateway contract that we're trying to mimic?

Comment (mustermeiszer) [reply pallets/liquidity-pools-gateway/axelar-gateway-precompile/src/lib.rs:249]: No! We have to keep it consistent to be easily callable by the EVM side.

Comment (cdamian) [reply pallets/liquidity-pools-gateway/axelar-gateway-precompile/src/lib.rs:249]: OK, thanks for clearing that out!

**Comment by mustermeiszer** [comment]
I can not approve as I created the PR. But looks all good to me!

**Review by cdamian — APPROVED** [review]

**Review by NunoAlexandre — APPROVED** [review]

**Pull request merged**